integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiServiceMigration.java

```diff
@@ -206,125 +217,113 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
         adminServerPodName, "",
         Paths.get(RESOURCE_DIR, "jms", "JmsSendReceiveClient.java"),
         Paths.get(destLocation)));
     runJavacInsidePod(adminServerPodName, domainNamespace, destLocation);
 
-    assertTrue(checkJmsServerRuntime("managed-server2"),
-         "JMSService@managed-server2 is on managed-server2 before migration");
+    assertTrue(checkJmsServerRuntime("ClusterJmsServer@managed-server2", 
+         "managed-server2"),
+        "ClusterJmsServer@managed-server2 is on managed-server2 before migration");
 
-    runJmsClientOnAdminPod("send",
-            "ClusterJmsServer@managed-server2@jms.testUniformQueue");
-
-    boolean psuccess = assertDoesNotThrow(() ->
-            scaleCluster(domainUid, domainNamespace, "cluster-1", 1),
-        String.format("replica patching to 1 failed for domain %s in namespace %s", domainUid, domainNamespace));
-    assertTrue(psuccess,
-        String.format("Cluster replica patching failed for domain %s in namespace %s", domainUid, domainNamespace));
-    checkPodDoesNotExist(managedServerPrefix + "2", domainUid, domainNamespace);
-    // Make sure the JMSService@managed-server2 is migrated to managed-server1
-    assertTrue(checkJmsServerRuntime("managed-server1"),
-            "JMSService@managed-server2 is NOT migrated to managed-server1");
-    runJmsClientOnAdminPod("receive",
-            "ClusterJmsServer@managed-server2@jms.testUniformQueue");
-  }
-
-  /**
-   * Verify JMS Service is migrated to an available active server.
-   * Here the JMS messages are stored in the JDBC store.
-   */
-  @Test
-  @DisplayName("Verify JMS Service migration with JDBCStore")
-  void testMiiJmsServiceMigrationWithJdbcStore() {
-
-    // Restart the managed server(2) if shutdown by previous test method
-    // Make sure that JMS server runtime JMSService@managed-server2 is
-    // hosted on managed server 'managed-server2'
-    restartManagedServer("managed-server2");
-    assertTrue(checkJmsServerRuntime("managed-server2"),
-         "JMSService@managed-server2 is on managed-server2 before migration");
+    assertTrue(checkJmsServerRuntime("JdbcJmsServer@managed-server2", 
+        "managed-server2"),
+        "JdbcJmsServer@managed-server2 is on managed-server2 before migration");
 
-    // build the standalone JMS Client on Admin pod after rolling restart
-    String destLocation = "/u01/JmsSendReceiveClient.java";
-    assertDoesNotThrow(() -> copyFileToPod(domainNamespace,
-        adminServerPodName, "",
-        Paths.get(RESOURCE_DIR, "jms", "JmsSendReceiveClient.java"),
-        Paths.get(destLocation)));
-    runJavacInsidePod(adminServerPodName, domainNamespace, destLocation);
+    assertTrue(checkJtaRecoveryServiceRuntime("managed-server2", 
+         "managed-server2", "true"), 
+         "JTARecoveryService@managed-server2 is not on managed-server2 before migration");
 
+    // Send persistent messages to both FileStore abd JDBCStore based
```

Comment:
FileStore abd JDBCStore
                ^^and

Reply:
done